hartshorn-assembly/checkstyle/suppressions.xml

```diff
@@ -5,5 +5,6 @@
         "https://checkstyle.org/dtds/suppressions_1_2.dtd">
 
 <suppressions>
+    <suppress files=".*Providers.java" checks="MissingJavadocMethod"/>
     <!-- No active suppressions -->
```

Comment:
Remove outdated comment, document why `.*Providers` is permitted to have undocumented methods